react-components/src/architecture/base/renderTarget/RevealRenderTarget.ts

```diff
@@ -237,11 +255,7 @@ export class RevealRenderTarget {
   // ==================================================
 
   public fitView(): boolean {
-    const boundingBox = this.clippedSceneBoundingBox;
-    if (boundingBox.isEmpty()) {
```

Comment:
Are we sure that this check is not need any more?

Reply:
In the viewer is now like this (I have change it, and merged to master)

```
fitCameraToBoundingBox(boundingBox: THREE.Box3, duration?: number, radiusFactor: number = 2): void {
    if (boundingBox.isEmpty()) {
      return;
    }
    this._activeCameraManager.fitCameraToBoundingBox(boundingBox, duration, radiusFactor);
  }

```